lib/cocoapods/generator/module_map.rb

```diff
@@ -42,8 +42,9 @@ def save_as(path)
       # @return [String]
       #
       def generate
+        header = target.requires_frameworks? ? "framework module" : "module"
```

Comment:
I think I know what you meant by calling that var `header`, but as that is used differently in modulemaps, it was irritating me. Let's call that var `module_declaration_qualifier`, which can have the value `"framework "` and keep the `module` in the string.